tests/ext/django/test_settings.py

```diff
@@ -0,0 +1,16 @@
+from unittest import mock
+
+import django
+from aws_xray_sdk import global_sdk_config
+from django.test import TestCase, override_settings
+from django.conf import settings
+from django.apps import apps
+
+from aws_xray_sdk.core import xray_recorder
+from aws_xray_sdk.core.sampling.sampler import LocalSampler
+
+
+class XRayConfigurationTestCase(TestCase):
+    def test_sampler_can_be_configured(self):
+        assert isinstance(settings.XRAY_RECORDER['SAMPLER'], LocalSampler)
+        assert isinstance(xray_recorder.sampler, LocalSampler)
```

Comment:
I would have loved to overwrite `settings.XRAY_RECORDER['SAMPLER']` here, to show the recorder working with different samplers.

But because `xray_recorder` is configured in the `ready` method of the app, and the `ready` method is only called once when the test suite is run, that's not easy to implement (cleanly) currently. (you can read more about it [here](https://stackoverflow.com/questions/31148172/django-override-setting-used-in-appconfig-ready-function), or verify with a `pdb.set_trace()` on the `ready` method)